collectd/files/df.conf

```diff
@@ -18,10 +18,14 @@ LoadPlugin df
       Device "{{ device }}"
 {%- endfor %}
 {%- if collectd_settings.plugins.df.MountPoint is defined and collectd_settings.plugins.df.MountPoint %}
-      MountPoint "{{ collectd_settings.plugins.df.MountPoint }}"
+    {%- for mountpoint in collectd_settings.plugins.df.MountPoints %}
```

Comment:
You check ``MountPoint`` (sic!) for existance, but use ``MountPoints`` for iterating.
I suppose, you should stuck on one parameter.